meta.yaml

```diff
@@ -3,28 +3,29 @@
 # ===================================================
 
 meta:
-  project: bb-buildingblock
+  project: bb-gitversion
   include:
   - dobi.yaml
```

Comment:
why is dobi.yaml included, or furthermore, why is dobi.sh a part of this repository?

Reply:
sorry, but i'm not sure, if i understand the question, but i try to answer:
dobi.sh is needed when you do further work on gitversion and you want to build, test and deploy the modifications.

Reply:
question was from a brick vs bb perspective point of view... but  i understand